specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/examples/Apps_Templates.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2018-09-01"
+  },
+  "responses": {
+    "200": {
+      "description": "Returns the application templates details",
+      "headers": {
+        "code": "200"
+      },
+      "body": {
```

Comment:
Looka t the other examples. I don't see why `id`, `location`, and other properties listed below would be in the response body for getting application templates.